pkg/api/openshiftcluster.go

```diff
@@ -27,6 +27,11 @@ type OpenShiftCluster struct {
 	Lock sync.Mutex `json:"-"`
 }
 
+// IsWimi checks whether a cluster is a Workload Identity cluster or Service Principal cluster
+func (oc *OpenShiftCluster) IsWimi() bool {
```

Comment:
What is "Wimi" - could we spell out "WorkloadIdenitity..." for readability please?

Reply:
@slawande2 , IMO you can also just use `IsWorkloadIdentity` instead. Reference of usage:- https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-console-supports-azure-sts-detection

Reply:
done, thank you for the suggestion.

Reply:
done, thank you for the suggestion.